src/storage/txn/scheduler.rs

```diff
@@ -616,7 +616,7 @@ fn process_read(
                     } else {
                         Ok(Some(Command::ResolveLock {
                             ctx: ctx.clone(),
-                            txn_status: txn_status.clone(),
+                            txn_status: txn_status.drain().collect(),
```

Comment:
If the original txn_status reused after this is called? If not the drain().collect() can become `mem::replace(txn_status, Default::default())`

Reply:
Thanks @arthurprs 

Seem we also use drain() + collect() in some places, could you help us to improve them?